tests/rspec_webui_tests.pm

```diff
@@ -27,6 +27,7 @@ sub test_flags() {
 }
 
 sub post_fail_hook {
+    upload_logs("/tmp/rspec_tests.txt");
```

Comment:
you should add the failok parameter :) `upload_logs('/tmp/rspec_tests.txt', failok => 1);`